File: EJERCICIO_1/EJERCICIOJAVA/dayTime.java
package EJERCICIOJAVA;

public class dayTime extends bonus {
    public dayTime( salaryBass salary, double Porcentage , int days) {
        super(salary, 1.25, days); 
    }

    public double calculateBonus() {
        return valueDay * percentage * days; 
    }
}

File: EJERCICIO_1/EJERCICIOSPHP/dayTime.php
<?php
class DayTime extends Bonus {
    public function __construct(SalaryBass $salaryBass, $percentage, $days) {
        parent::__construct($salaryBass, 1.25, $days); 
    }

    public function calculateBonus() {
        return $this->valueDay * $this->percentage * $this->days; 
    }
}

?>